internal/server/data/migrator/migrator_test.go

```diff
@@ -362,46 +355,25 @@ func runDBTests(t *testing.T, fn func(t *testing.T, db DB)) {
 	databases := []dbDriver{}
 
 	if pg := database.PostgresDriver(t, "_migrator"); pg != nil {
-		databases = append(databases, dbDriver{dialect: "postgres", driver: pg.Dialector})
+		databases = append(databases, dbDriver{dialect: "postgres", dsn: pg.DSN})
 	}
 
 	for _, database := range databases {
 		// Ensure defers are not stacked up for each DB
-		t.Run(database.driver.Name(), func(t *testing.T) {
-			db, err := gorm.Open(database.driver, &gorm.Config{})
+		t.Run(database.dialect, func(t *testing.T) {
+			db, err := sql.Open("pgx", database.dsn)
```

Comment:
This is where we switch from gorm to stdlib for opening the DB (only in the migrator tests for now).